modules/DSA-02_project-1.yml

```diff
@@ -26,13 +26,13 @@ screens:
               id: DSA-02-new-content-verification
               success-message: "Great job - you successfully updated the file."
               failure-message: "It looks like you haven't updated the file yet."
-        - description: "Make sure the change shows up under Pull Request #5. If it does, make a comment in the PR about the update."
+        - description: "Create a Pull Request to merge the `add-gitignore` branch into `master`."
           id: DSA-02-pr-comment
```

Comment:
Right, the ID is completely irrelevant in terms of what the validation does. It does have some implications for how this is imported over old data, and so it's easier for now to keep the same ID.